exercise/map.js

```diff
@@ -0,0 +1,56 @@
+/*
+## Map
+
+### Instructions
+
+Create an array named 'arr2' that contains the values of 'arr1'
+with the following modifications applied to each of the values:
+1) They have to be multiplied by 2
+2) Then the results have to be subtracted by 4
+3) finally, the negative values have to be replaced by null
+then do the same with arrRandom1 by returning arrRandom2
+The use of `for` and `while` is forbidden for this exercise
```

Comment:
I think this should be a functions, and even more that we should have a few different use cases.
Not just 1 use of map.

I also don't thing this is a very demonstrative usecase of map.

Reply:
reworked the subject to ask for functions.